src/actions/formActions.js

```diff
@@ -10,24 +12,36 @@ export const formActions = {
   TOGGLE_SHOW_FORM,
 };
 
-export const toggleShowForm = formName => ({
-  type: TOGGLE_SHOW_FORM,
-  formName,
-});
+export const addChip = (router, name, input, limit) => {
```

Comment:
Mostly I mean for this component, a form shouldn't always just modify query string or something. This form might want to do that, but future forms may not want to. If we could find a way to localize this behavior in the tablefilterform component instead of the form component, I think it would be more reusable